gadget-container/entrypoint.sh

```diff
@@ -104,7 +104,9 @@ if [ "$ID" = "rhcos" ] && [ ! -d "/usr/src/kernels/$KERNEL" ]; then
     curl --max-time 40 -fsSLo $RPMDIR/$RPM $REPO/$RPM || rm -f $RPMDIR/$RPM
 
     if test -f $RPMDIR/$RPM; then
-      cd $RPMDIR && rpm2cpio $RPM | cpio --quiet -i
+      pushd $RPMDIR
+      rpm2cpio $RPM | cpio --quiet -i
+      popd
```

Comment:
> The previous logic was changing the directory to a path and then removing it, which was causing issues when calling "getcwd" in BCC python scripts.
>
Oops, I didn't know it could be a problem. Good to know for next time. Thanks!

Reply:
I was a bit surprising for me too and took some time to realize what was going on.